pkg/security/module/server.go

```diff
@@ -68,15 +73,49 @@ func (e *EventServer) SendEvent(rule *eval.Rule, event eval.Event) {
 	case e.msgs <- msg:
 		break
 	default:
+		// Update stats
+		eventsStat, ok := e.droppedEvents[rule.ID]
+		if ok {
+			atomic.AddInt64(eventsStat, 1)
+		}
 		// Do not wait for the channel to free up, we don't want to delay the processing pipeline further
 		log.Warnf("the event server channel is full, an event of ID %v was dropped", msg.RuleID)
```

Comment:
should we convert this to trace now ? otherwise we will have the log sent to the log backend ?

Reply:
👍